src/ElectricPokemon.java

```diff
@@ -0,0 +1,68 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+// Variabelen
+    private static final String type = "Electric";
+    List<String> attacks = Arrays.asList("thunderPunch", "electroBall", "thunder", "voltTackle");
+
+// Constructor
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super(type, name, level, hp, food, sound);
+    }
+
+
+// Methodes
+public void attackFactor(Pokemon name, Pokemon enemy, int attackHp, String attackName) {
```

Comment:
Heel netjes dat je deze logica in een aparte functie hebt gezet, zo hoef je niet 4 keer over dezelfde code te typen